### PR TITLE
simplify bootstrap, split to bootstrap and bootstrap_ci

### PR DESCRIPTION
**Pull request by ildarusmanov**
#154

### CODE REVIEW

**Comment by vast** [comment]
@vladimirtemnikov, @VladimirMikhailov, what's the status of this one? Is it related to #224?

**Comment by VladimirMikhailov** [comment]
@vast I guess, it doesn't relate to the #224, it's more about splitting `bin/bootstrap` into two versions: for ci and dev setups.

Status: We can make review and merge or close it =). Cap.

File: bin/bootstrap [hunk 1]
```diff
@@ -54,10 +51,6 @@ class Bootstrap
     system_with_note('./bundle_cache download ; echo 1')
```
Comment (vast) [inline bin/bootstrap]: is this "bundle_cache" thing used somewhere?

File: bin/bootstrap [hunk 2]
```diff
@@ -37,10 +37,7 @@ class Bootstrap
   end
 
   def install_gems
-    without = is_ci? ? 'development' : ''
-    path = is_ci? ? 'vendor/bundle' : '.bundle'
-
-    system_with_note("bundle install --path #{path} --without production staging #{without}")
+    system_with_note("bundle install --path .bundle --without production staging")
```
Comment (vast) [inline bin/bootstrap]: what if I have to run application locally in production environment?

**Comment by vast** [comment]
:+1:

**Pull request merged**